### ISSUE TITLE
if I give a bad expression in prefixlenfilter I get an exception

### ISSUE THREAD

**Issue by jopietsch** [netenglabs/suzieq#115]
**Describe the bug**
rather than an error and a help message, I get an exception

**To Reproduce**
```
jpiet> route show namespace=single-attach_bgp_numbered prefixlen='>=24 || <=30'                                                                                                                           Error: 1Error running command: Python keyword not valid identifier in numexpr query (<unknown>, line 1)
------------------------------------------------------------
Traceback (most recent call last):
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/src/python-nubia/nubia/internal/cmdbase.py", line 372, in run_interactive
    ret = fn(**args_dict)
  File "/tmp/pycharm_project_304/suzieq/suzieq/cli/sqcmds/RouteCmd.py", line 118, in show
    prefixlen=prefixlen,
  File "/tmp/pycharm_project_304/suzieq/suzieq/sqobjects/basicobj.py", line 109, in get
    return self.engine_obj.get(**kwargs)
  File "/tmp/pycharm_project_304/suzieq/suzieq/engines/pandas/routes.py", line 19, in get
    return df.query(f'prefix.ipnet.prefixlen {prefixlen}')
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/frame.py", line 3231, in query
    res = self.eval(expr, **kwargs)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/frame.py", line 3346, in eval
    return _eval(expr, inplace=inplace, **kwargs)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/computation/eval.py", line 332, in eval
    parsed_expr = Expr(expr, engine=engine, parser=parser, env=env)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 764, in __init__
    self.terms = self.parse()
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 781, in parse
    return self._visitor.visit(self.expr)
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 371, in visit
    raise e
  File "/home/jpiet/.local/share/virtualenvs/suzieq-zI29E9ll/lib/python3.7/site-packages/pandas/core/computation/expr.py", line 367, in visit
    node = ast.fix_missing_locations(ast.parse(clean))
  File "/usr/lib/python3.7/ast.py", line 35, in parse
    return compile(source, filename, mode, PyCF_ONLY_AST)
  File "<unknown>", line 1
    prefix .ipnet .prefixlen >=24 or or <=30
                                      ^
SyntaxError: Python keyword not valid identifier in numexpr query
------------------------------------------------------------
```

**Comment by ddutt** [comment]
Exception doesn't occur, but the help message is not clear either.
```
ddutt> route show namespace=nxos prefixlen='>=24 || <=30'
                                                                             error
0  ERROR: Python keyword not valid identifier in numexpr query (<unknown>, line 1)
```
A workaround is to use query-str as follows:
```
ddutt> route show namespace=nxos columns='hostname vrf prefix prefixlen nexthopIps' query-str='prefixlen >=24 and prefixlen <=30'                                                                                                                                                                                    Error: 1
      hostname           vrf            prefix  prefixlen                      nexthopIps
0     dcedge01       default     10.255.2.0/24         24                              []
1     dcedge01       default    169.254.0.0/24         24                              []
2     dcedge01       default     172.16.1.0/24         24                 [169.254.127.3]
3     dcedge01       default     172.16.2.0/24         24                 [169.254.127.3]
...
31      exit02      evpn-vrf    169.254.0.0/24         24                 [169.254.253.6]
32      exit02      evpn-vrf  169.254.253.4/30         30                 [169.254.253.5]
...
```

**Comment by ddutt** [comment]
Fixed in 0.16.0 alpha
